web/html/src/manager/state/recurring-states-list.js

```diff
@@ -116,25 +62,86 @@ class RecurringStatesList extends React.Component {
                                 <h3>Schedules</h3>
                             </div>
                         </div>
-                        <div className="panel-body">
-                            <table className="table table-striped">
-                                <thead>
-                                <tr>
-                                    <th>{t("Active")}</th>
-                                    <th className="text-center">{t("Schedule Name")}</th>
-                                    <th className="text-center">{t("Frequency")}</th>
-                                    {targetType}
-                                    <th className="text-right">{t("Actions")}</th>
-                                </tr>
-                                </thead>
-                                {this.tableBody()}
-                            </table>
-                            <DeleteDialog
-                                id="delete-modal"
-                                title={t("Delete Recurring State Schedule")}
-                                content={<span>{t("Are you sure you want to delete the selected item?")}</span>}
-                                onConfirm={() => this.props.onDelete(this.state.itemToDelete)}
-                                onClosePopUp={() => this.selectToDelete(null)}
+                        <div>
+                            <Table
+                                data={this.props.data}
+                                identifier={action => action.recurringActionId}
+                                //initialSortColumnKey="scheduleName"
+                                initialItemsPerPage={this.props.disableCreate ? userPrefPageSize : 0}
+                                emptyText={t("No schedules created." + (this.props.disableCreate ? "" : " Use Create to add a schedule."))}
+                            >
+                                <Column
+                                    columnKey="active"
+                                    header={t('Active')}
+                                    cell={(row, criteria) => {return (
+                                        <Toggler
+                                            value={row.active}
+                                            className="btn"
+                                            handler={() => this.props.onToggleActive(row)}
+                                        />
+                                    );
+                                    }}
+                                />
+                                <Column
+                                    columnClass="text-center"
+                                    headerClass="text-center"
+                                    columnKey="scheduleName"
+                                    header={t('Schedule Name')}
+                                    cell={(row, criteria) => row.scheduleName}
```

Comment:
This comment is valid for all the `Column`s here: I think we can omit the `criteria` parameter as it is never used, am I missing something?

Reply:
No you are not, I will just remove them